### PR TITLE
add cve values to openvas csv parser

### PR DESCRIPTION
**Pull request by manuel-sommer**
See https://github.com/DefectDojo/django-DefectDojo/discussions/9780

### CODE REVIEW

**Review by mtesauro — APPROVED** [review]
Approved

**Review by Maffooch — CHANGES_REQUESTED** [review]
The finding `cve` field will be deprecated in the near future. Please use the `vulnerability_id` field instead. It allows for having multiple CVEs on a single finding in the case of `CveColumnMappingStrategy`

File: dojo/tools/openvas/csv_parser.py [hunk 1]
```diff
@@ -71,6 +72,31 @@ def map_column_value(self, finding, column_value):
...
 
+class CveColumnMappingStrategy(ColumnMappingStrategy):
+    def __init__(self):
+        self.mapped_column = "cves"
+        super(CveColumnMappingStrategy, self).__init__()
+
+    def map_column_value(self, finding, column_value):
+        if "," in column_value:
+            finding.description += "\n**All CVEs:** " + str(column_value)
+            finding.unsaved_vulnerability_ids.append(column_value.split(",")[0])
```
Comment (Maffooch) [inline dojo/tools/openvas/csv_parser.py:83]: If there are multiple CVEs in this list, we should probably save more than just the first one

File: unittests/tools/test_openvas_parser.py [hunk 2]
```diff
@@ -47,6 +47,40 @@ def test_openvas_csv_many_vuln(self):
...
+            parser = OpenVASParser()
+            findings = parser.get_findings(f, test)
+            for finding in findings:
+                for endpoint in finding.unsaved_endpoints:
+                    endpoint.clean()
+            self.assertEqual(13, len(findings))
+            finding = findings[2]
+            self.assertEqual("Apache HTTP Server Detection Consolidation", finding.title)
+            self.assertEqual("Info", finding.severity)
+            self.assertEqual(finding.unsaved_vulnerability_ids[0], "")
```
Comment (Maffooch) [inline unittests/tools/test_openvas_parser.py:83]: This seems like unexpected behavior. Why would we want an empty string to be here?

**Review by Maffooch — CHANGES_REQUESTED** [review]

**Review by Maffooch — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Pull request merged**